AzureKeyVaultUtils/Program.cs

```diff
@@ -0,0 +1,87 @@
+﻿using AzureKeyVaultUtils.Helpers;
+using AzureKeyVaultUtils.Models;
+using CommandLine;
+using Serilog;
+using System.Text.Json;
+
+namespace AzureKeyVaultUtils
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Configuring logger
+            Log.Logger = new LoggerConfiguration()
+            .WriteTo.Console()
+            .CreateLogger();
+
+            // Handle input arguments
+            HandleCommandLineArguments(args);
+
+            // Flush the logger
+            Log.CloseAndFlush();
+        }
+
+        private static void HandleCommandLineArguments(string[] args)
+        {
+            Parser.Default.ParseArguments<Options>(args)
+            .WithParsed(options =>
+            {
+                if (options.Help)
+                {
+                    DisplayHelpDesc();
+                }
+                else if (options.Info)
+                {
+                    Log.Information(Constants.Message.InfoMessage);
+                }
+                else
+                {
+                    Log.Information($"Process started.");
+                    KeyVaultHelper vaultHelper = new KeyVaultHelper(options.Url!);
+                    switch (options.Operation)
+                    {
+                        case OperationEnum.Insert:
+                            //Performs insertion of key-value pairs into the Azure Key Vault
+                            Log.Information($"Reading the json file.");
+                            string jsonContent = File.ReadAllText(options.FilePath!);
+                            JsonDocument jsonDocument = JsonDocument.Parse(jsonContent);
+                            Dictionary<string, string> secrets = new Dictionary<string, string>();
+                            JsonHelper.GetSecretsFromJson(jsonDocument.RootElement, secrets);
```

Comment:
why you have to pass dictionary? directly the method can return dictionary right.